code/datums/station_traits/neutral_traits.dm

```diff
@@ -64,3 +64,19 @@
 /datum/station_trait/hangover/revert()
 	. = ..()
 	SSjobs.drunken_spawning = FALSE
+
+/datum/station_trait/triple_ai
+	name = "AI Triumvirate"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 1
+	show_in_report = TRUE
+	report_message = "As part of Operation Magi, your station has been instated with three Nanotrasen Artificial Intelligence models. Try not to break them."
```

Comment:
```suggestion
	report_message = "As part of Operation Magi, your station has been equipped with three Nanotrasen Artificial Intelligence models. Please try not to break them."
```
Just a suggestion, I think this would flow better. More of a nit than anything.